### ISSUE TITLE
does not work if stderr is not a TTY

### ISSUE THREAD

**Issue by derhuerst** [jonschlinkert/window-size#8]
```js
console.log(require('window-size'))
```

Running this works fine if `stderr` is a TTY (not piped). If I run with `node example.js ^/dev/null` (fish syntax), both `width` and `height` are `undefined`.

afaict [line 18](https://github.com/jonschlinkert/window-size/blob/0dc3fc0/index.js#L18) is the problem.

**Comment by jonschlinkert** [comment]
> afaict line 18 is the problem.

Line 18 is a conditional, not a problem. Since you didn't suggest an alternative, I'm guessing this is a feature request?

**Comment by derhuerst** [comment]
> afaict line 18 is the problem.

Ok, let me rephrase it: The problem lies within line 18.

> Since you didn't suggest an alternative, I'm guessing this is a feature request?

I will make a PR to just remove it, except if there's a specific reason why this check exists.
